charts/argo/values.yaml

```diff
@@ -57,6 +57,9 @@ controller:
   #  spec:
   #    ttlStrategy:
   #      secondsAfterCompletion: 84600
+  scaling: {}
```

Comment:
I am not sure the `scaling` secion is 100% nescassary and keeping it directly under `controller` would suffice.

Reply:
Yep makes sense. Was thinking of making it "thematic" but it probably makes for unnecessary nesting.